inference-engine/ie_bridges/python/wheel/setup.py

```diff
@@ -302,6 +306,9 @@ def get_package_dir(install_cfg):
 if os.path.exists(package_license):
     copyfile(package_license, "LICENSE")
 
+packages = find_packages(','.join(get_dir_list(PY_INSTALL_CFG)))
+package_data = {}
```

Comment:
Is it ok that previously `package_data` was equal to result of `find_data_files(get_dir_list(LIB_INSTALL_CFG))` and currently it is always empty dictionary?

Reply:
dev package doesn't  include ie_libs and can be empty, but for runtime package package_data will be added in generate_package()

Reply:
excluded from this PR